test/integration/cni/service_connectivity_test.go

```diff
@@ -140,6 +155,15 @@ var _ = Describe("[CANARY] test service connectivity", func() {
 
 		err = f.K8sResourceManagers.DeploymentManager().DeleteAndWaitTillDeploymentIsDeleted(deployment)
 		Expect(err).ToNot(HaveOccurred())
+
+		//sleep for cooldown period
+		time.Sleep(5 * time.Second)
+
+		k8sUtils.UpdateEnvVarOnDaemonSetAndWaitUntilReady(f, "aws-node", "kube-system",
```

Comment:
nit: I think we can remove this and unset the variable in `AfterSuite`. This variable being set for other test cases should not matter